### ISSUE TITLE
Issues regarding Roberta

### ISSUE THREAD

**Issue by jacklxc** [thunlp/KernelGAT#17]
Hi, your paper reported Roberta-large performance, but the code only provides BERT-base. Is there anything special to do when replacing BERT with Roberta?

I tried to replace your `BertForSequenceEncoder` with `Huggingface AutoModel` and it worked fine on "bert-base-uncased" and "allenai/scibert_scivocab_uncased".

However, when I choose "Roberta-base", `nan` loss occurs after a few batches.
```
[06-09-2020 16:51:56] INFO: Epoch: 0, Step: 10, Loss: 1.0664133727550507
[06-09-2020 16:51:59] INFO: Epoch: 0, Step: 20, Loss: 1.0591850370168685
[06-09-2020 16:52:01] INFO: Epoch: 0, Step: 30, Loss: nan
[06-09-2020 16:52:03] INFO: Epoch: 0, Step: 40, Loss: nan
```
Even worse, when I choose "roberta-large" with correct bert_dim=1024, "RuntimeError: CUDA error: an illegal memory access was encountered" occurs after the first 10 steps.

Do you have any idea why this happens? Is it related to how you designed the KGAT module after BERT? I noticed that Roberta model has an extra `<\s>` than BERT after the first `<\s>`. Is the error related to this?

Thank you.

**Comment by EdwardZH** [comment]
For RoBERTa models, the learning rate should set to 3e-5 and add segment labels to the model. 

For the RuntimeError, I guess it is the CUDANN bug. Please refer to https://github.com/pytorch/pytorch/issues/21819.

**Comment by EdwardZH** [comment]
In addition, I fix a bug to keep the model training stable.

**Comment by jacklxc** [comment]
Can you release the exact code for running Roberta-large?

I have been using Roberta-large frequently without any CUDA issue.

**Comment by EdwardZH** [comment]
I can release them. We have run  KGAT (RoBERTa large) frequently, no same issues occur. The pytorch present this issue and I have no idea if you do not trust it.

**Comment by EdwardZH** [comment]
I have updated the RoBERTa large codes and checkpoints. I will close this issue.

**Comment by jacklxc** [comment]
Thank you for the release. Can you also post the exact environment that you used for this work? It seems that the CUDA issue is related to some specific PyTorch version.

**Comment by jacklxc** [comment]
FYI I finally found the solution to the CUDA issue and nan loss issue that I encountered. In this line https://github.com/thunlp/KernelGAT/blob/79abd35c82e97e1b57d50f00b2368a28add8f86c/kgat/models.py#L129
Roberta model (roberta-base, roberta-large) only takes two inputs: `inp_tensor, msk_tensor`. An extra argument will cause the program to crash. 

I'm not sure this is a new change by Huggingface, our you forgot to fix when running with Roberta, but anyway this simple fix worked for me.
